ArduCopter/Copter.cpp

```diff
@@ -143,7 +143,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #endif
     FAST_TASK(Log_Video_Stabilisation),
 
-    SCHED_TASK(rc_loop,              100,    130,  3),
+    SCHED_TASK(rc_loop,              250,    130,  3),
```

Comment:
@andyp1per so in the end this is the only functional change since the uart buffer cannot be decreased below 512?

Reply:
Also 3 frames is a functional change